src/modules/Tab/Tab.js

```diff
@@ -96,19 +102,42 @@ class Tab extends Component {
     })
   }
 
+  renderColumn(columnProps) {
+    return GridColumn.create(columnProps)
+  }
+
+  renderVertical(menu) {
+    const { grid, panes } = this.props
+    const { activeIndex } = this.state
+
+    return (
+      <Grid>
+        {menu.props.tabular !== 'right' && this.renderColumn({ width: grid.tabWidth, children: menu })}
+        {this.renderColumn({
+          width: grid.segmentWidth,
+          children: _.invoke(_.get(panes, `[${activeIndex}]`), 'render', this.props),
```

Comment:
This line:
```js
_.invoke(_.get(panes, `[${activeIndex}]`), 'render', this.props)
```
is duplicated in the main `render()` method.  Let's pull this line into a method like `renderActivePane()` to contain it.